models/mock_model.py

```diff
@@ -118,6 +144,9 @@ def get_state_of_charge(self) -> Optional[int]:
     def get_lv_battery(self) -> Optional[int]:
         return self.current_data[6]
 
+    def get_current(self) -> Optional[float]:
+        return self.current_data[7]
```

Comment:
do we actually need current_data in the mock model? every use of it in this file is pointless

Reply:
I thought it would make it easier to line up with raspberry model